rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java

```diff
@@ -676,9 +680,19 @@ public BlockResultDTO eth_getBlockByNumber(String bnOrId, Boolean fullTransactio
 
     @Override
     public TransactionResultDTO eth_getTransactionByHash(String transactionHash) {
+        if(!HexUtils.hasHexPrefix(transactionHash)) {
```

Comment:
I think the aim of this task is to make rskj compatible with GETH, but if RSKJ were supporting hashes without the `0x` prefix, then we shouldn't be more restrictive now. and allow then.

Reply:
I see, got it. I did it like this because the ticket provided a document that shows this difference so I assumed they should make the same validation.